### ISSUE TITLE
Comment by Stephen Smartt

### ISSUE THREAD

**Issue by fedhere** [fedhere/Explosive-Physics-Fast-Transients-White-Paper-Cadence#12]
Comment by Stephen Smartt
3.11 says it does not need any more time. I don’t understand that, it must, by definition require either 50% more time each night. Therefore the sky area covered each night is reduced by 33%.  (@MelissaGraham  I think it is fine as we say it but what do you think?)

Interesting idea, and will be interesting to see the results of the OpSim runs. The numbers of transients with significant rise/fall within 1-2hrs will be very low.  You need to estimate this, and can just use the volumetric SN and KN rate, to do a sanity check calculation.  For example forDT2=1.5hrs compared to next night (DT2=24hrs), means a factor ~16 reduction  in the number of objects detected.  It’s not clear that the Fig 1 objects actually need 1.5hrs, nor if they are high enough sky density to justify aiming for detection.

@pcowpert @igorandreoni @tylerapritchard @rbiswas4 @MelissaGraham can we do some back of the envelope calculation here?  He is right...

**Comment by MelissaGraham** [comment]
I also think 3.11 is fine as is; this is a WFD modification, and as is already said, a shuffling of the visits and not an addition/subtraction of time. I think 3.11 is more for DDF and Mini-Surveys.

For the second paragraph, that is more complicated, no comment yet...

**Comment by rbiswas4** [comment]
I thought we were going to say that a WFD modification of this kind would be ideal, but we could also think of this as a mini-survey. In this case, we do need extra time. WFD already provides pairs at 30 mins. So, there the request is to make sure the pairs are in different bands. (As g count ~ 80, r count ~ 184  in 10 years per field), the number of g-r pairs per field in 10 yrs is limited by the g count. ie ~ 80. 
If we ask for the extra visit in either g or r 2 hours later as part of a mini-survey. This is 80 * 2.3 * 10^3 which is ~ 80 * 10^{-3} or 0.8 percent of the survey.

**Comment by MelissaGraham** [comment]
Oh OK then maybe add a blurb about how "if this is implemented as a mini-survey, it would mean X extra time" or something like that? I did notice also that in Section 1, Item 2, it says this "could be a WFD or Mini-Survey or DDF" and it confused me a bit. I was going to ask about it. Will the white paper provide specific requests for implementing each option in an OpSim? Shouldn't this go in Section 6 "Request for New OpSim Runs" ... oh, which is missing from our template. But the DESC WFD white paper has such a Section 6. ???  @fedhere ?

**Comment by rbiswas4** [comment]
@MelissaGraham  I agree! think we should have a 'Requests' section to ask for simulations that do explore these kinds of options.

Is it useful to have a redshift limit of the follow up instrument, and a rough number of how many we can hope to follow up in a year?

**Comment by MelissaGraham** [comment]
I'd say we would probably be fine leaving a quantification of follow-up activities out of it, there's science to be done with light curves alone -- but also if you did add it, that would be fine too.
